docs/specs/bind-unbind-workflow.md

```diff
@@ -0,0 +1,22 @@
+## Bind / Unbind New Work flow
+
+The goal of this new work flow is to give more control over the actions that the database takes on bind and unbind to the APB author. This means that we want the apb to be launched on on bind and unbind. 
```

Comment:
The new part is that we have no way of telling the apb how to talk to the database. See comment below on what we need to change. I can change the name to update or enhance if you would like?